newsletter-december-2022.rst

```diff
@@ -0,0 +1,121 @@
+.. post:: Dec 1, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - December 2022
+========================================
+
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-december-2022>`.
+
+
+News and updates
+----------------
+
+The latest updates from our team:
+
+- 🐘️ We started *tooting* in the Fediverse (Mastodon): `@readthedocs@fosstodon.org <https://fosstodon.org/@readthedocs>`_
+- 🎬️ Visit the upcoming landing page `about.readthedocs.com <https://about.readthedocs.com/>`_ and `let us know <mailto:hello@readthedocs.org>`_ know if you have comments or ideas for it.
+- ⚙️ The *entire* build process can now be overridden (if you need it). :doc:`See the Announcement </build-customization>`.
+
+  - 💡️ ...this is useful if you want to publish outputs generated from a framework that isn't supported by default. We wrote some examples for :ref:`Pelican <readthedocs:build-customization:Pelican>` and :ref:`Docsify <readthedocs:build-customization:Docsify>`.
+  - 💡️ ...you can also *extend* the build process for instance you can :ref:`skip a build <readthedocs:build-customization:Cancel build based on a condition>`.
+
+- ⬇️ Sphinx 6 is coming soon, more details :ref:`further down <sphinx6_is_coming>`.
+- 🛳️ :doc:`Server-Side Search API v3 <readthedocs:server-side-search/index>` has been released.
+- ⏳️ We organized a large refactor of our user documentation to comply with the `Diátaxis methodology framework <https://diataxis.fr>`_.
+  So far, we broke it down into `73 tasks and counting <https://github.com/readthedocs/readthedocs.org/issues?q=is%3Aissue++diataxis+iteration+>`_.
+  The tasks are big and small.
+- ✅️ We added :doc:`readthedocs:unofficial-projects`.
+- ✅️ We added `an additional auditing feature <https://github.com/readthedocs/readthedocs.org/pull/9607>`_,
+  whereby invitations are added to the Security Log.
+  The feature is available for users of Read the Docs for Business.
+- ✅️ We found, fixed and disclosed a security issue,
+  `XSS: Allow serving of arbitrary HTML files from main domain <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-98pf-gfh3-x3mp>`_.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+.. _sphinx6_is_coming:
```

Comment:
This ref is not needed since we use `autosectionlabel`, I guess?

Reply:
No, not in this project. Maybe it's because it's so old :)

We can open an issue to do that.

Reply:
-1 on adding more work to our blog. We should focus on migrating if anything.